docs/content/doc/help/faq.en-us.md

```diff
@@ -403,3 +403,8 @@ gitea doctor recreate-table
 ```
 
 It is highly recommended to back-up your database before running these commands.
+
+
+## Why are tabs/indents wrong when viewing files
+
+If you are using Cloudflare, turn off the auto-minify option in the dashboard.
```

Comment:
Might want to mention that this option is under Speed -> Optimization:
![image](https://user-images.githubusercontent.com/25481501/180123656-762db7d1-24a2-4539-8d20-8b1eee15e19e.png)

And then specifically the HTML part:
![image](https://user-images.githubusercontent.com/25481501/180123726-97cea789-4088-48cb-87ed-d5e913bf3c61.png)